<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenetreAjoutRecette</class>
 <widget class="QDialog" name="FenetreAjoutRecette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: darkgray;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="3" column="3">
      <widget class="QLineEdit" name="le_duree"/>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Durée de préparation :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type de recette :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QTextEdit" name="te_etapes"/>
     </item>
     <item row="6" column="3">
      <widget class="QComboBox" name="cb_image"/>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Etapes de préparation :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ingrédients</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_nom"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Image :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableIngredients">
         <property name="styleSheet">
          <string notr="true">QHeaderView::section {
    background-color:gray; /*qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);*/
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row/>
         <column>
          <property name="text">
           <string>Ingrédients</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantités</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Unité</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="tbRetirerIng">
           <property name="text">
            <string> - </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbAjouterIng">
           <property name="text">
            <string> + </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cb_type">
       <item>
        <property name="text">
         <string>Autre</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Boisson</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Légume</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fruit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Viande</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poisson</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fromage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Entrée</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dessert</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_nom</tabstop>
  <tabstop>tableIngredients</tabstop>
  <tabstop>tbRetirerIng</tabstop>
  <tabstop>tbAjouterIng</tabstop>
  <tabstop>le_duree</tabstop>
  <tabstop>te_etapes</tabstop>
  <tabstop>cb_image</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FenetreAjoutRecette</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FenetreAjoutRecette</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
